src/main/java/Duke/DukeFunction/DukePrinter.java

```diff
@@ -0,0 +1,49 @@
+package Duke.DukeFunction;
+
+public class DukePrinter {
```

Comment:
I like that you have reorganised the things to print into a different `.java`. However, it would be better to standardise, and have all your print statements in this `.java`. Currently, I do still spot a few print statements in other files.